docs/docs/configuration/sql-templating.mdx

```diff
@@ -292,6 +292,41 @@ Here's a concrete example:
   WHERE country_code = 'US'
   ```
 
+**Guest User Attributes**
```

Comment:
Could be great to clarify, explain or link to a place in the docs as to how guest user attributes are set in the first place (?)